tests/functional/ConfigTest.php

```diff
@@ -83,4 +83,52 @@ public function testConfigExportImportStatusExistingConfig()
             $this->assertContains('unish existing', $page['front'], 'Existing config was successfully imported during site:install.');
         }
     }
+
+    public function testConfigImport()
+    {
+        $options = [
+            'include' => __DIR__,
+        ];
+        $this->setupModulesForTests(['woot'], Path::join(__DIR__, 'resources/modules/d8'));
+        $this->drush('pm-enable', ['woot'], $options);
+
+        // Export the configuration.
+        $this->drush('config:export');
+
+        $root = $this->webroot();
+
+        // Introduce a new service in the Woot module that depends on a service
+        // in the Devel module (which is not yet enabled).
+        $filename = Path::join($root, 'modules/unish/woot/woot.services.yml');
+        $serviceDefinition = <<<YAML_FRAGMENT
+  woot.depending_service:
+    class: Drupal\woot\DependingService
+    arguments: ['@devel.dumper']
+YAML_FRAGMENT;
+        file_put_contents($filename, $serviceDefinition, FILE_APPEND);
+
+        $filename = Path::join($root, 'modules/unish/woot/woot.info.yml');
+        $moduleDependency = <<<YAML_FRAGMENT
+dependencies:
+  - devel
+YAML_FRAGMENT;
+        file_put_contents($filename, $moduleDependency, FILE_APPEND);
+
+        // Add the 'devel' module in core.extension.yml.
+        $extensionFile = $this->getConfigSyncDir() . '/core.extension.yml';
+        $this->assertFileExists($extensionFile);
+        $extension = Yaml::decode(file_get_contents($extensionFile));
+        $extension['module']['devel'] = 0;
+        require_once $root . "/core/includes/module.inc";
+        $extension['module'] = module_config_sort($extension['module']);
+        file_put_contents($extensionFile, Yaml::encode($extension));
+
+        $this->drush('config:import');
+    }
```

Comment:
This is not actually testing that config import works. It's making an assumption that if the command is successful it's successful but it doesn't actually properly test that a module has been enabled and it in the container.modules parameter.